app/Http/Transformers/ReportbackTransformer.php

```diff
@@ -21,8 +21,7 @@ public function transform(Post $post)
             'id' => $post->id,
             'status' => $post->status,
             'caption' => $post->caption,
-            // Add link to review reportback item in Rogue here once that page exists
-            // 'uri' => 'link_goes_here'
+            'uri' => url(config('services.phoenix.uri') . '/api/v1/reportbacks/'.$post->id, ['absolute' => true]),
```

Comment:
yeah with Rogue on I don't think anything will live there (I thiiiink Phoenix would try to grab the reportback that has an rbid that matches `$post->id`??)